serverless/installing_serverless/serverless-install-config-options.adoc

```diff
@@ -6,82 +6,11 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
-This guide provides information for cluster administrators about advanced installation configuration options for Knative Serving and Knative Eventing.
+This guide provides information for cluster administrators about advanced installation configuration options for Knative Serving.
+// TODO: Add Eventing once there's something substantial to add.
```

Comment:
Knative Eventing also has this form by default. Only registry is possible to set. I don't know the meaning of that setting, though.

![Zrzut ekranu z 2020-07-17 16-30-40](https://user-images.githubusercontent.com/540893/87797621-41c8c300-c84b-11ea-9e0d-0535d41ccec2.png)

Reply:
Related to the other discussion above - not sure if these fields are supported or should be removed in the UI. Will follow up.